### PR TITLE
Release notes and documentation for Forms 13.1.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Added release notes and documentation for Forms 13.1.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [X] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms 13.1

## Deadline (if relevant)

Can be reviewed now but please don't merge yet as I still need to add the dates of the release candidates, and want to make sure docs are ready to go first.  Thanks.

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @AndyButland thank you for the PR 🙌 

We will make sure to review it as soon as we can 🙏

File: 13/umbraco-forms/developer/configuration/README.md [hunk 1]
```diff
@@ -491,7 +499,13 @@ You can obtain both of these values after signing up to create a ReCaptcha key h
...
 This setting defines the domain from which the client-side assets for using the reCAPTCHA service are requested.
 
-Valid options are `www.google.com` (the default) or `www.recaptcha.net`. You may want to use the latter for control of which domains are setting cookies on your site. [Read more at the reCAPTCHA documentation](https://developers.google.com/recaptcha/docs/faq#does-recaptcha-use-cookies).
+Valid options are `Google` (the default) or `Recaptcha`. You may want to use the latter for control of which domains are setting cookies on your site. [Read more at the reCAPTCHA documentation](https://developers.google.com/recaptcha/docs/faq#does-recaptcha-use-cookies).
+
+#### VerificationUrl
+
+By default the server-side validation of the reCAPTCHA response is sent to Google's servers at `https://www.google.com/recaptcha/api/siteverify`.
+
+Some customers with a locked down production environment cannot configure the firewall to allow these requests and instead use a proxy server. They can use this setting to configure the URL to their proxy server, which will relay the request to and response from Google.
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/configuration/README.md:508]:
```suggestion
Some customers with a locked-down production environment cannot configure the firewall to allow these requests and instead use a proxy server. They can use this setting to configure the URL to their proxy server, which will relay the request to and response from Google.
```

File: 13/umbraco-forms/developer/configuration/README.md [hunk 2]
```diff
@@ -491,7 +499,13 @@ You can obtain both of these values after signing up to create a ReCaptcha key h
 
 This setting defines the domain from which the client-side assets for using the reCAPTCHA service are requested.
 
-Valid options are `www.google.com` (the default) or `www.recaptcha.net`. You may want to use the latter for control of which domains are setting cookies on your site. [Read more at the reCAPTCHA documentation](https://developers.google.com/recaptcha/docs/faq#does-recaptcha-use-cookies).
+Valid options are `Google` (the default) or `Recaptcha`. You may want to use the latter for control of which domains are setting cookies on your site. [Read more at the reCAPTCHA documentation](https://developers.google.com/recaptcha/docs/faq#does-recaptcha-use-cookies).
+
+#### VerificationUrl
+
+By default the server-side validation of the reCAPTCHA response is sent to Google's servers at `https://www.google.com/recaptcha/api/siteverify`.
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/configuration/README.md:506]:
```suggestion
By default, the server-side validation of the reCAPTCHA response is sent to Google's servers at `https://www.google.com/recaptcha/api/siteverify`.
```

File: 13/umbraco-forms/developer/configuration/README.md [hunk 3]
```diff
@@ -223,6 +225,12 @@ Take care to not hide any settings that are required for the particular field or
 
 The default value and read-only settings apply to most setting types. There is an exception for complex ones where a default string value isn't appropriate. An example of one of these is the field mapper used in the "Send to URL" workflow.
 
+### MandatoryFieldsetLegends
+
+When creating a form with Umbraco Forms, adding captions to the groups for fields is optional. To follow accessibility best practice, these fields should be completed. When they are, the group of fields are presented within a `<fieldset>` element that has a populated `<legend>`.
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/configuration/README.md:230]:
```suggestion
When creating a form with Umbraco Forms, adding captions to the groups for fields is optional. To follow accessibility best practices, these fields should be completed. When they are, the group of fields are presented within a `<fieldset>` element that has a populated `<legend>`.
```

File: 13/umbraco-forms/developer/webhooks.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Webhooks
+
+Umbraco Forms will register events for workflow operations that you can use with [Umbraco webhooks](https://docs.umbraco.com/umbraco-cms/reference/webhooks).
+
+Workflows are operations that you can associated with form submission, approval or rejections actions. You can use these where you need to notify external systems of the success or failure of a workflow.
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/webhooks.md:5]:
```suggestion
Workflows are operations that you can associate with form submission, approval, or rejection actions. You can use these where you need to notify external systems of the success or failure of a workflow.
```

File: 13/umbraco-forms/developer/webhooks.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+Via the Umbraco _Settings > Webhooks_ dashboard you can configure webhooks to respond to workflows.
```
Comment (eshanrnh) [inline 13/umbraco-forms/developer/webhooks.md:7]:
```suggestion
On the Umbraco **Settings** > **Webhooks** dashboard, you can configure webhooks to respond to workflows.
```

File: 13/umbraco-forms/release-notes.md [hunk 6]
```diff
@@ -17,6 +17,25 @@ If you are upgrading to a new major version, you can find information about the
 
 This section contains the release notes for Umbraco Forms 13 including all changes for this version.
 
+#### [**13.1.0**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.1.0) **(March 5th 2024)**
+
+With the introduction of webhooks with Umbraco 13, we've made an update to Forms to allow workflow execution events to trigger webhooks. Workflows are operations that you can associated with form submission, approval or rejections actions. You can use these where you need to notify external systems of the success or failure of a workflow. Read more on [Umbraco Forms wehbooks](./developer/webhooks.md).
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:22]:
```suggestion
With the introduction of webhooks in Umbraco 13, we've made an update to Forms to allow workflow execution events to trigger webhooks. Workflows are operations that you can associate with form submission, approval, or rejection actions. You can use these where you need to notify external systems of the success or failure of a workflow. Read more on [Umbraco Forms Wehbooks](./developer/webhooks.md).
```

File: 13/umbraco-forms/release-notes.md [hunk 7]
```diff
@@ -17,6 +17,25 @@ If you are upgrading to a new major version, you can find information about the
...
+
+We've added an update that will help customers using Forms within locked down production environments.
+
+And there are a couple of further additions to improve performance and accessibility of the product.
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:26]:
```suggestion
And there are a couple of further additions to improve the performance and accessibility of the product.
```

File: 13/umbraco-forms/release-notes.md [hunk 8]
```diff
@@ -17,6 +17,25 @@ If you are upgrading to a new major version, you can find information about the
 
 This section contains the release notes for Umbraco Forms 13 including all changes for this version.
 
+#### [**13.1.0**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.1.0) **(March 5th 2024)**
+
+With the introduction of webhooks with Umbraco 13, we've made an update to Forms to allow workflow execution events to trigger webhooks. Workflows are operations that you can associated with form submission, approval or rejections actions. You can use these where you need to notify external systems of the success or failure of a workflow. Read more on [Umbraco Forms wehbooks](./developer/webhooks.md).
+
+We've added an update that will help customers using Forms within locked down production environments.
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:24]: This seems like an incomplete sentence or some words are missing from it 🤔

**Review by eshanrnh — COMMENTED** [review]
Looks good to me, @AndyButland 🙌 I've added couple of minor suggestions here and there.

Please let me know the date when it should be merged 💪

File: 13/umbraco-forms/release-notes.md (hunk 8 above)
Comment (AndyButland) [reply 13/umbraco-forms/release-notes.md:24]: Not really, it's just calling out one of the features in the list below.  It's quite specific, so didn't want to go into detail.

**Comment by AndyButland** [comment]
Thanks @eshanrnh,I  have accepted suggestions, and this can go live at any time so long as it's before 5th March.

**Review by eshanrnh — APPROVED** [review]
Merging it in 🙌

**Pull request merged**